### ISSUE TITLE
Generated Mixing recipes should use fluids when ingredient includes #forge:milk or honey bottles

### ISSUE THREAD

**Issue by Xaidee** [PssbleTrngle/SliceAndDice#11]
### Describe the feature you have in mind.

I suggest having it check if the cooking recipe has an item ingredient of "#forge:milk" or "minecraft:honey_bottle" for it to swap those out for their respective fluids for cooking with the basin. i.e. #forge:milk -> 250mB minecraft:milk, minecraft:honey_bottle -> 250mB create:honey

### Does this solve a particular problem?

As it currently is, it isn't a problem but it'd make more sense to utilise fluids when it comes to basins like the create mod does with several of its own recipes

**Comment by PssbleTrngle** [comment]
I've actually attempted this, because I would like it too, but the problem is that some of these fluid ingredients are given as an item tag (ex: milk bottles/buckets as `#forge:milk`). At the time of recipe generation I do not have access to the items belonging to the tags yet.

For now I have added two specific recipes which use these ingredient as hard-coded recipes with the fluid ingredient, but I would like to make it work too.
